desired_distance: 0.30
communication_radius: 1.0
alpha: 2.0
k_fi: 0.5
k_d: 3.0
